package.json

```diff
@@ -14,10 +14,8 @@
     "@ethersproject/address": "^5.4.0",
     "@ethersproject/bignumber": "^5.4.1",
     "@ethersproject/bytes": "^5.4.0",
-    "@ethersproject/hash": "^5.4.0",
     "@ethersproject/providers": "^5.4.3",
     "@ethersproject/solidity": "^5.4.0",
-    "@ethersproject/strings": "^5.4.0",
```

Comment:
This is still used here:
https://github.com/snapshot-labs/snapshot/blob/d91b6fdd180720b909b0de2207f100f864912b15/src/components/Token.vue#L3
https://github.com/snapshot-labs/snapshot/blob/d91b6fdd180720b909b0de2207f100f864912b15/src/components/Modal/ClearDelegate.vue#L7
https://github.com/snapshot-labs/snapshot/blob/d91b6fdd180720b909b0de2207f100f864912b15/src/views/Delegate.vue#L7